src/services/home.ts

```diff
@@ -1,6 +1,7 @@
 import moment from "moment";
 import _ from "lodash";
 const dateFor = moment().format("YYYY-MM-DD");
+import API from "../utils/api";
```

Comment:
Please revert this file change as the PR title and task do not mention this change or else create a task and add in the title.

@deepali-chavhan